website_and_docs/data/sponsors.yml

```diff
@@ -10,10 +10,10 @@ selenium:
     - logo: "/images/sponsors/saucelabs.png"
       url: "http://saucelabs.com"
       name: "Sauce Labs"
-    # Sponsorship start date: June 24, 2015
-    - logo: "/images/sponsors/experitest.png"
+    # Sponsorship start date: June 24, 2015, updated July 2023
```

Comment:
Can we have start date too in here (or) can we just by Month wise ?

Any thoughts on this @diemol !

Reply:
This is a @SeleniumHQ/selenium-plc matter. So not sure. However, they said we could merge the PR.